xml/art_sle_ha_nfs_quick.xml

```diff
@@ -491,7 +491,8 @@ include /etc/drbd.d/*.res;</screen>
 <screen>&prompt.crm.conf;<command>primitive</command> nfsserver \
   systemd:nfs-server \
   op monitor interval="30s"
-&prompt.crm.conf;<command>clone</command> cl-nfsserver nfsserver
+&prompt.crm.conf;<command>clone</command> cl-nfsserver nfsserver \
+   meta interleave=true
```

Comment:
Exactly, thanks for the fix!

Reply:
Just for the records. I've committed the fix for:

* develop (SLE15/SLE15SP1): https://github.com/SUSE/doc-sleha/commit/dc8d2462ebd804
* maintenance/SLEHA12SP3:   https://github.com/SUSE/doc-sleha/commit/432ee31d9a0950
  (that will also be part for SP4)